pkg/services/alerting/notifier.go

```diff
@@ -24,9 +24,38 @@ var newImageUploaderProvider = func() (imguploader.ImageUploader, error) {
 type NotifierPlugin struct {
 	Type            string          `json:"type"`
 	Name            string          `json:"name"`
+	Heading         string          `json:"heading"`
 	Description     string          `json:"description"`
+	Info            string          `json:"info"`
 	OptionsTemplate string          `json:"optionsTemplate"`
 	Factory         NotifierFactory `json:"-"`
+	Options         []Option        `json:"options"`
+}
+
+// Option holds information about options specific for the NotifierPlugin.
+type Option struct {
```

Comment:
Since this is a larger `alerting` package, is this more specifically a `NotifierOption` ?